dbbackup/management/commands/mediabackup.py

```diff
@@ -113,7 +137,7 @@ def get_backup_file_list(self):
         if server_name:
             server_name = '-%s' % server_name
 
-        media_re = re.compile(r'^%s%s-(.*)\.media\.tar\.gz' % (self.get_databasename(), server_name))
+        media_re = re.compile(r'^%s%s-(.*)\.media\.tar' % (self.get_databasename(), server_name))
```

Comment:
At least, instead of removing the .gz extension which is because the file is compressed with Gzip (which is also default option), it should be there and perhaps with a condition if kwargs.get('compress') is open!?

Renaming files is a very bad idea because people have implemented behaviors on their systems that depend on how dbbackup works... we dont wanna risk breaking people's backup systems!